CHANGELOG.md

```diff
@@ -7,6 +7,73 @@ pull request if there was one.
 
 Current
```

Comment:
`Current` is explicitly unversioned because we couldn't specify a full version triplet (ie. `x.y.z`, since the `.z` portion will advance with every push to `master`. 

The README _does_ have a dynamic badge that points at the most recent version in Bintray, so that (I think) should satisfy some of the usefulness you're looking for here, but out of curiosity, what use do you think including a version number here would provide? It's quite likely that there are uses I've not thought of 😄!